decidim-accountability/app/views/decidim/participatory_processes/participatory_process_groups/_highlighted_results.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if results.any? %>
```

Comment:
Can this partial be moved inside `decidim-accountability`?

Reply:
I think that I don't understand your comment, can you reword? The partial is already inside the `decidim-accountability` module.

This is done in the same way that the meetings PR: https://github.com/decidim/decidim/pull/2713

Reply:
Yes, the file is in the correct location 👍

Reply:
My bad, I didn't expand the path to the file 😅